test/functional/test_framework/script.py

```diff
@@ -87,7 +91,7 @@ def decode_op_n(self):
 
     def is_small_int(self):
         """Return true if the op pushes a small integer to the stack"""
-        if 0x51 <= self <= 0x60 or self == 0:
+        if 0x51 <= self <= 0x60 or self == 0 or self == -1:
```

Comment:
`self` is never -1, the check should be for `0x4f`. Maybe we could rewrite this using named opcodes instead of bytes:
```suggestion
        if self == OP_0 or OP_1 <= self <= OP_16 or self == OP_1NEGATE:
```

Reply:
Done in 6cf08dd2d532766f1b7063f40e7ab25f8ee67e45